misc/graylog.conf

```diff
@@ -169,9 +169,47 @@ rest_listen_uri = http://127.0.0.1:9000/api/
 # The size of the thread pool used exclusively for serving the web interface.
 #web_thread_pool_size = 16
 
+# List of elasticsearch hosts Graylog should connect to.
```

Comment:
Aesthetic nitpick: The name is "Elasticsearch" (not "elasticsearch" or "ElasticSearch").